curriculum/challenges/english/25-front-end-development/workshop-plant-nursery-catalog/6734ab61e916486c80c490d1.md

```diff
@@ -0,0 +1,81 @@
+---
+id: 6734ab61e916486c80c490d1
+title: Step 11
+challengeType: 0
+dashedName: step-11
+---
+
+# --description--
+
+When the value associated with a key is an object, the `get` method will return a reference to that object. You can use that reference to modify the object inside the map.
+
+To test it, set the `small` property of `getBallerina` to an integer different from `20`. Then, look at the developer console to check that the value has changed in your catalog.
+
+# --hints--
+
+You should set the `small` property of `getBallerina` to an integer different from `20`.
+
+```js
+const small = catalog.get(ballerina).small;
+assert.match(__helpers.removeJSComments(code), /getBallerina\s*(?:\.\s*small|\[\s*("|')small\1\s*\])\s*=\s*\d+/);
+assert.notEqual(small, 20);
+assert.isTrue(Number.isInteger(small));
```

Comment:
```suggestion
assert.isNumber(small);
```
Why not something like this? 

https://www.chaijs.com/api/assert/#method_isnumber

Reply:
If that's something we're interested in , I can point out all the spots where we can change it.